chains/solana/contracts/programs/ccip-offramp/src/instructions/v1/config.rs

```diff
@@ -1,34 +1,16 @@
 use crate::state::SourceChainConfig;
 
-pub fn get_on_ramps(config: &SourceChainConfig) -> Vec<&[u8]> {
-    let first_item = trim_trailing_zeros(&config.on_ramp[0]);
-    let second_item = trim_trailing_zeros(&config.on_ramp[1]);
-
-    let mut on_ramps: Vec<&[u8]> = Vec::new();
-
-    // filter out empty values
-    if !first_item.is_empty() {
-        on_ramps.push(first_item);
-    }
-    if !second_item.is_empty() {
-        on_ramps.push(second_item);
-    }
-
-    on_ramps
+pub fn get_on_ramps(config: &SourceChainConfig) -> impl Iterator<Item = &[u8; 64]> {
```

Comment:
can you add a comment here on why we are doing this?

Reply:
Yep, will do 🙂